### DISCUSSION TITLE
Arbitrary connectivity

### DISCUSSION THREAD

**Discussion by bsureshkrishna** [rafaqz/Cellular.jl#245] [General]
Hello, very naively, and at a high level, is this package potentially a good solution if one is interested not in spread that happens locally over a map, with units likely to influence some of their neighbours more than far away units, but instead is interested in more arbitrary, long-distance connectivity (like in a sheet-like neural network of neurons ?)

Thanks !

**Comment by rafaqz** [comment]
Sorry I missed this, I hadn't used the discussions before.

You can do long distance connectivity (https://github.com/cesaraustralia/Dispersal.jl) with various kinds of jumps.

It can make sense if they are all on a grid, and you also need to work with local neighborhoods. But if you are only doing long distance jumps, you may want to do that on a graph.

This human driven dispersal example calculates a truncated tail  of destinations with a distrubiont and umps to them randomly: https://github.com/cesaraustralia/Dispersal.jl/blob/main/src/human.jl.

Its a pretty fast way to do that and worked will in our simulations:

![](https://raw.githubusercontent.com/cesaraustralia/packagegifs/master/SWD_USA.gif)
